.github/workflows/tests_full.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests Full
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:
+    types: [review_requested, ready_for_review, auto_merge_enabled]
+
+jobs:
+  build:
+    runs-on: arc-runner-set
+    strategy:
+      matrix:
+        python-version: ["3.11"] # ["3.10", "3.11"]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get install git -y
+        python -m pip install torch
+        python -m pip install --upgrade pip setuptools wheel
+        export FLASH_ATTENTION_SKIP_CUDA_BUILD=TRUE
+        python -m pip install -e .[tests]
+    - name: Run tests
+      run: |
+        pytest
+        # sh tests/run_all_tests.sh 0 1
+    - name: Upload coverage data to coveralls.io
+      run: |
+        python -m pip install coveralls[toml]
+        COVERALLS_INPUT=.coverage_reports/.coverage coveralls --service=github
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is throwing some errors in the workflow:
```
Successfully installed coveralls-4.0.1 docopt-0.6.2
Submitting coverage to coveralls.io...
Failed collecting git data. Are you running coveralls inside a git repository? Is git installed?
Traceback (most recent call last):
  File "/home/runner/_work/_tool/Python/3.11.9/x64/lib/python3.11/site-packages/coveralls/git.py", line 16, in run_command
    cmd = subprocess.run(
          ^^^^^^^^^^^^^^^
  File "/home/runner/_work/_tool/Python/3.11.9/x64/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['git', '--no-pager', 'log', '-1', '--pretty=format:%H']' returned non-zero exit status 128.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/runner/_work/_tool/Python/3.11.9/x64/lib/python3.11/site-packages/coveralls/git.py", line 88, in git_info
    'id': gitlog('%H'),
          ^^^^^^^^^^^^
  File "/home/runner/_work/_tool/Python/3.11.9/x64/lib/python3.11/site-packages/coveralls/git.py", line 30, in gitlog
    return run_command(
           ^^^^^^^^^^^^
  File "/home/runner/_work/_tool/Python/3.11.9/x64/lib/python3.11/site-packages/coveralls/git.py", line 22, in run_command
    raise CoverallsException(
coveralls.exception.CoverallsException: Command '['git', '--no-pager', 'log', '-1', '--pretty=format:%H']' returned non-zero exit status 128.
STDOUT: b''
STDERR: b'fatal: not a git repository (or any of the parent directories): .git\n'
```
But I wonder if it's necessary to submit a coverage report from this workflow at all, as it seems redundant, given that the same code coverage is already checked in the Test workflow...